librz/core/cmd/cmd_print.c

```diff
@@ -4953,6 +4953,67 @@ static void cmd_pxr(RzCore *core, int len, int mode, int wordsize, const char *a
 	}
 }
 
+static void core_print_2bpp_row(const ut8 *buf, bool useColor) {
+	const char *symbols = "#=-.";
+	for (ut32 i = 0, c = 0; i < 8; i++) {
+		if (buf[1] & ((1 << 7) >> i)) {
+			c = 2;
+		}
+		if (buf[0] & ((1 << 7) >> i)) {
+			c++;
+		}
+		if (useColor) {
+			char *color = "";
+			switch (c) {
+			case 0:
+				color = Color_BGWHITE;
+				break;
+			case 1:
+				color = Color_BGRED;
+				break;
+			case 2:
+				color = Color_BGBLUE;
+				break;
+			case 3:
+				color = Color_BGBLACK;
+				break;
+			}
+			rz_cons_printf("%s  ", color);
+		} else {
+			const char ch = symbols[c % 4];
+			rz_cons_printf("%c%c", ch, ch);
+		}
+		c = 0;
+	}
+}
+
+static void core_print_2bpp_tiles(RzCore *core, ut32 tiles) {
+	const ut8 *buf = core->block;
+	bool useColor = rz_config_get_i(core->config, "scr.color") > 0;
+	for (ut32 i = 0; i < 8; i++) {
+		for (ut32 r = 0; r < tiles; r++) {
+			core_print_2bpp_row(buf + 2 * i + r * 16, useColor);
+		}
+		if (useColor) {
+			rz_cons_printf(Color_RESET "\n");
+		} else {
+			rz_cons_printf("\n");
+		}
+	}
+}
+
+static inline void core_print_raw_buffer(const ut8 *buffer, ut32 length, ut32 option, ut32 max_wrap_len) {
```

Comment:
```suggestion
static void core_print_raw_buffer(const ut8 *buffer, ut32 length, ut32 option, ut32 max_wrap_len) {
```
no need for inline, it doesn't do anything here.

Reply:
i have set it as inline on purpose to avoid duplication since this code is so small.

Reply:
It doesn't work that way AFAIK. `static` is enough to make compiler do its magic. `inline` is useful only in `.h` files (again, AFAIK). @thestr4ng3r cc

Reply:
ok, i will remove it then.